wannierberri/__Data_K.py

```diff
@@ -47,26 +48,19 @@ def __init__(self,system,dK,grid,Kpoint=None,npar=0,fftlib='fftw',npar_k=1 ):
         self.ksep = system.ksep
         self.use_wcc_phase=system.use_wcc_phase
         if self.use_wcc_phase:
-            self.wannier_centres_reduced=system.wannier_centres_reduced
-            self.wannier_centres_cart=system.wannier_centres_cart
+            # if we consistently "ignore" the factor exp(ik*(tj-ti)) bothin the Wannier Hamiltonian, and in the  
+            # thransformation to Hamiltonian gauge, the result will be correct again.
+            # We only need wcc to evaluate "," derivatives (to be used in cRvec_wcc)
```

Comment:
This is the essential part of the new implementation. Could you make it explicit here that when calculating the derivatives we multiply R if False and R+ti - tj if True, and this choice is done inside cRvec_wcc?Btw, the name crvec_wcc is somewhat confusing because it is just crvec if use_wcc=false.